tool/cpufreqctl

```diff
@@ -93,6 +94,29 @@ not_supported ()
     EXIT_CODE=9
 }
 
+# echo min/max/avg/rnd of a space-separated list of numbers
+compute_min_max_avg_rnd ()
+{
+    measures=($1)
+    nb_measures=${#measures[@]}
```

Comment:
:see_no_evil: this is bashism and may or may not work on distros like Alpine or Ubuntu (they use ash and dash). You can use `shellcheck -a` to lint against these things.

Reply:
Don't tell me my `sh` is, in fact, `bash`....

Reply:
it is on Arch but that's not for granted

Reply:
It should be a symlink

Reply:
The whole sh vs. bash vs. dash compatibility thing is a *deep* rabbit hole. Yet another good reason to use GJS instead :see_no_evil:

Reply:
Nooooo you're right! I'm correcting that right away.

Reply:
It's not that easy. POSIX sh can't handle arrays...

Reply:
Another option is to require `bash`... ? But yeah, I spent way to much time on it, GJS *is* the right way.

Reply:
Nah, we can't require bash as this would rule out the whole Ubuntu land.